### ISSUE TITLE
"500 Server Error" while fetching node information

### ISSUE THREAD

**Issue by palette-dev** [pterodactyl/panel#4149]
### Current Behavior

The panel makes continous requests to `admin/nodes/view/1/system-information` to get the currently looked at node's information, whicht is the expected behaivor.
The problem is that the request only returns a 500 error with the following response:
```json
{
    "errors": [
        {
            "code": "UnexpectedValueException",
            "status": "500",
            "detail": "An unexpected error was encountered while processing this request, please try again."
        }
    ]
}
```
I've tried looking at the code for a possible sorce for the error, but I haven't found anything useful.
The error also doesn't produce any logs for some reason (believe me when I say that I tried finding any).

### Expected Behavior

The Panel's server returns the servers information normaly

### Steps to Reproduce

I've freshly installed the panel via the docker image using docker-compose with the following settings:
```yaml
version: '3.8'
x-common:
  database:
    &db-environment
    MYSQL_PASSWORD: &db-password "[REDACTED]"
    MYSQL_ROOT_PASSWORD: "[REDACTED]"
  panel:
    &panel-environment
    APP_URL: "[REDACTED]"
    APP_TIMEZONE: "[REDACTED]"
    APP_SERVICE_AUTHOR: "[REDACTED]"
  mail:
    &mail-environment
    MAIL_FROM: "[REDACTED]"
    MAIL_DRIVER: "smtp"
    MAIL_HOST: "mail"
    MAIL_PORT: "1025"
    MAIL_USERNAME: ""
    MAIL_PASSWORD: ""
    MAIL_ENCRYPTION: "true"

services:
  database:
    image: mariadb:10.7
    restart: always
    networks:
      - default
    command: --default-authentication-plugin=mysql_native_password
    volumes:
      - "${DATA_DIR}/database:/var/lib/mysql"
    environment:
      <<: *db-environment
      MYSQL_DATABASE: "panel"
      MYSQL_USER: "pterodactyl"

  cache:
    image: redis:alpine
    restart: always
    networks:
      - default
    volumes:
      - "cache:/data/"

  panel:
    image: ghcr.io/pterodactyl/panel:v1.8.1
    restart: always
    networks:
      vlan-bridge:
        ipv4_address: 10.0.2.3
      default:
    links:
      - database
      - cache
    volumes:
      - "${DATA_DIR}/panel/var/:/app/var/"
      - "${DATA_DIR}/panel/logs/:/app/storage/logs"
      - "${DATA_DIR}/panel/nginx/:/etc/nginx/conf.d/"
    environment:
      <<: [*panel-environment, *mail-environment]
      DB_PASSWORD: *db-password
      APP_ENV: "production"
      APP_ENVIRONMENT_ONLY: "false"
      CACHE_DRIVER: "redis"
      SESSION_DRIVER: "redis"
      QUEUE_DRIVER: "redis"
      REDIS_HOST: "cache"
      DB_HOST: "database"
      TRUSTED_PROXIES: "*"

networks:
  vlan-bridge:
    external: true
  default:
    external: false
    ipam:
      config:
        - subnet: 172.20.0.0/16

volumes:
  cache:
```
The node is a fresh install of Debian Bullseye and Wings (no other modifications)

I've added an location and the node to the panel, and the system health check works in the node list, but when I look at the node the error occures continously

### Panel Version

1.8.1

### Wings Version

1.6.4

### Is there an existing issue for this?

- [X] I have searched the existing issues before opening this issue.
- [X] I have provided all relevant details, including the specific game and Docker images I am using if this issue is related to running a server.
- [X] I have checked in the Discord server and believe this is a bug with the software, and not a configuration issue with my specific system.

**Comment by parkervcp** [comment]
Have you pulled the panel and wings logs yet for this? So have all you have is "500 error" which can be any number of things.

**Comment by palette-dev** [comment]
I've already tried getting any logs out of the Panel (as I already mentioned in the issue), but this hasn't been successful yet.
Also, I checked the incomming trafic for my Wings server, and there isn't really anything from the Panel server, so that's probably not the issue (running with the debug option also doesn't show anything).
It also can't be my firewall, as all ports that pterodactyl uses are opened.
Nothing produces any logs anywhere, so I'm pretty clueless on where the issue is.
